src/Microsoft.AspNetCore.Rewrite/IISUrlRewriteOptionsExtensions.cs

```diff
@@ -65,5 +65,17 @@ public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, TextR
 
             return options;
         }
+
+        /// <summary>
+        /// Adds a rule to the current rules.
+        /// </summary>
+        /// <param name="options">The <see cref="RewriteOptions"/>.</param>
+        /// <param name="rule">A <see cref="IISUrlRewriteRule"/> to be added to the current rules.</param>
+        /// <returns>The Rewrite options.</returns>
+        public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, IISUrlRewriteRule rule)
```

Comment:
build.cmd is lazy, sometimes you need to delete the cached files before it will download new ones. `git clean -xdf`